proto/build.gradle.kts

```diff
@@ -7,13 +7,15 @@ import com.google.protobuf.gradle.protoc
 
 sourceSets.main {
     proto.srcDirs("../third_party/proto/")
-    java.srcDirs("build/generated/source/proto/main/java")
 }
 
 dependencies {
-    api(Libraries.ProtobufJava)
-    api(Libraries.GrpcStub)
+//    protobuf(files("cosmWasm-v0.17.0.tar.gz"))
```

Comment:
Do we need this?

Reply:
This is prep for when all protos will be included in the zip from provenance. I commented vs removing so I dont forget how to do it